views/pulsar/v2/helpers/form.html.twig

```diff
@@ -573,7 +573,6 @@ class   | string | A space separated list of class names
 
 #}
 {% macro end(options = {}) %}
-{% spaceless %}
```

Comment:
I'd have thought so too, but seems on a combination of removing both solves it. I'm no expert in Twig stuff though!